playbook/app/entrypoints/playbook.js

```diff
@@ -0,0 +1,8 @@
+import 'lazysizes/plugins/attrchange/ls.attrchange'
+import 'lazysizes'
+
+export * from '../javascript/kits'
+export * from '../javascript/theme'
+export * from '../javascript/dashboard'
+export * from '../javascript/plugins'
+export * from '../javascript/tokens'
```

Comment:
Notice some house cleaning here. Instead of having a huge file with the kitchen sink in it, we now have smaller units of code which were categorized based on the comments left in the original. Feel free to rename how you see fit.